apps/docs/components/content/pre.tsx

```diff
@@ -23,7 +23,11 @@ export function Pre(props: DetailedHTMLProps<HTMLAttributes<HTMLPreElement>, HTM
 				// ref={container}
 				className={cn(
 					'bg-zinc-900 text-sm text-white shadow md:rounded-xl overflow-x-auto px-5 md:px-4 py-4 font-medium',
-					'[td_&]:bg-zinc-200 dark:[td_&]:bg-zinc-800 [td_&]:shadow-none [td_&]:rounded-lg [td_&]:p-0 [td_&]:px-1.5'
+					'[td_&]:bg-zinc-200 dark:[td_&]:bg-zinc-800 [td_&]:shadow-none [td_&]:rounded-lg [td_&]:p-0 [td_&]:px-1.5',
+					// this rule enables inverted dark/light mode inside of table cells
+					'[&_*]:text-[--shiki-dark]',
```

Comment:
i don't love all these `*` rules - if there's a tighter alternative that would be great, but i also won't lose sleep over it :P

Reply:
it could be `span` instead of `*` i guess